library_generation/test/utilities_unit_tests.py

```diff
@@ -373,6 +375,7 @@ def test_prepare_repo_split_repo_success(self):
         self.assertEqual("output", Path(repo_config.output_folder).name)
         library_path = sorted([Path(key).name for key in repo_config.libraries])
         self.assertEqual(["misc"], library_path)
+        shutil.rmtree(repo_config.output_folder, ignore_errors=True)
```

Comment:
Why we can not use `__remove_generated_files` here?

Reply:
`__remove_generated_files` is used to delete `.repo-metadata.json`, `.OwlBot-hermetic.yaml` and `owlbot.py` and I changed the method name to `__remove_prerequisite_files`.